Dockerfile

```diff
@@ -15,8 +15,19 @@
 ###################
 # Conditions
 ###################
-FROM quay.io/cloudservices/io-gathering-conditions:qa AS conditions
 
+ARG CONDITIONS_VERSION="0.1.0"
+
+FROM registry.redhat.io/rhel8/go-toolset:1.16 AS conditions
+
+RUN git clone --branch $CONDITIONS_VERSION https://github.com/RedHatInsights/insights-operator-gathering-conditions
```

Comment:
Use `--depth 1` to download only the necessary files corresponding to that tag.

Reply:
is not it `--bare` ?

Reply:
that's even lighter, true!

Reply:
with --bare i have this
```
[dpensier@localhost temp]$ git clone --bare --branch $CONDITIONS_VERSION https://github.com/RedHatInsights/insights-operator-gathering-conditions
Cloning into bare repository 'insights-operator-gathering-conditions.git'...
remote: Enumerating objects: 85, done.
remote: Counting objects: 100% (85/85), done.
remote: Compressing objects: 100% (50/50), done.
remote: Total 85 (delta 30), reused 73 (delta 24), pack-reused 0
Receiving objects: 100% (85/85), 15.03 KiB | 2.50 MiB/s, done.
Resolving deltas: 100% (30/30), done.
[dpensier@localhost temp]$ ls
insights-operator-gathering-conditions.git
[dpensier@localhost temp]$ cd insights-operator-gathering-conditions.git/
[dpensier@localhost insights-operator-gathering-conditions.git]$ ls
branches  config  description  HEAD  hooks  info  objects  packed-refs  refs
[dpensier@localhost insights-operator-gathering-conditions.git]$ cd branches/
[dpensier@localhost branches]$ ls
[dpensier@localhost branches]$ cd ..
[dpensier@localhost insights-operator-gathering-conditions.git]$ 
```

Reply:
This is what I got:
```
❯ git clone --depth 1 --branch "0.1.0" https://github.com/RedHatInsights/insights-operator-gathering-conditions
...
❯ cd insights-operator-gathering-conditions
❯ ls
build_deploy.sh  build.sh  conditions  Dockerfile  pr_check.sh  README.md  requirements.txt  validate.py

```